src/main/java/com/teamsfinder/userwriteservice/user/model/User.java

```diff
@@ -0,0 +1,40 @@
+package com.teamsfinder.userwriteservice.user.model;
+
+ import com.teamsfinder.userwriteservice.tag.model.Tag;
+ import lombok.Getter;
+ import lombok.NoArgsConstructor;
+ import lombok.Setter;
+
+ import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
```

Comment:
jak nie masz konstruktora żadnego zdefiniowanego to masz zapewniony przez jave konstruktor bez argumentów więc to zbędne